src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -112,7 +112,7 @@ public void getSalaryInfoByTwoMonths() throws Exception {
     @Test
     public void testClassExists() {
         try {
-            Class.forName("core.basesyntax.exception.IllegalDateParametersException");
+            Class.forName("core.basesyntax.core.basesyntax.exception.IllegalDateParametersException");
```

Comment:
the same here) let's revert it to the previous version

Reply:
same as upper, yes I modified test, but then fixed it back :)

Reply:
Or I didn't pay attention and changed it.